src/cs-graphics/internal/tiny_gltf_helper.hpp

```diff
@@ -92,9 +92,11 @@ auto find_material_parameter(
     return def;
   } else {
     auto const& parameter = it->second;
-    T           value;
+    T           value{};
     for (size_t i = 0; i < std::min(size_t(value.length()), parameter.number_array.size()); ++i) {
+#pragma warning(disable: 4267 4244)
       value[i] = parameter.number_array[i];
+#pragma warning(default: 4267 4244)
     }
```

Comment:
I haven't tested it, but this should work as T is always a glm type:

```suggestion
    for (size_t i = 0; i < std::min(size_t(value.length()), parameter.number_array.size()); ++i) {
      value[i] = static_cast<typename T::value_type>(parameter.number_array[i]);
    }
```